src/features/events/l10n/messageIds.ts

```diff
@@ -35,7 +35,9 @@ export default makeMessages('feat.events', {
     dialogTitle: m('Move event'),
     error: m('Error: Could not move the event to the selected project'),
     moveButtonLabel: m('Move'),
```

Comment:
Note to remove this message after updating to keep it in the messages of the dialog instead of sending it in (typescript unfortunately doesn't help out by catching if a message is unused).